guides/common/modules/proc_installing-openscap-plugin.adoc

```diff
@@ -1,32 +1,32 @@
 [id="Installing_the_OpenSCAP_Plugin_{context}"]
 = Installing the OpenSCAP Plug-in
 
-You can install and enable the OpenSCAP plug-in to generate OpenSCAP compliance reports.
-The OpenSCAP plug-in consists of the main OpenSCAP plug-in itself, the OpenSCAP smart proxy plug-in, and the OpenSCAP Hammer CLI plug-in.
+You can install and enable the OpenSCAP plug-in to generate SCAP compliance reports.
+The OpenSCAP plug-in consists of the OpenSCAP {Project} plug-in, the OpenSCAP Hammer CLI plug-in, the OpenSCAP {SmartProxy} plug-in, and modules for policy deployment.
 
 .Procedure
 . Install the OpenSCAP plug-in on your {ProjectServer}:
 +
 [options="nowrap" subs="+quotes,attributes"]
 ----
-# {foreman-installer} --enable-foreman-plugin-openscap --enable-foreman-proxy-plugin-openscap --foreman-proxy-plugin-openscap-puppet-module true
+# {foreman-installer} \
+--enable-foreman-plugin-openscap \
+--enable-foreman-cli-openscap \
```

Comment:
`satellite-cli` requires the CLI package, so
```suggestion
ifndef::satellite[]
--enable-foreman-cli-openscap \
endif::[]
```

(`satellite-cli` is pulled in by `satellite`)

Interestingly it doesn't require the scap plugin itself.

Reply:
This module is not included for satellite, so I'm resolving this.